wf.json

```diff
@@ -10,4 +10,5 @@
   "git": "https://github.com/senx/warp10-ext-polyglot",
   "tags": "warpscript, warp 10, groovy, lua, r, python, ruby, javascript",
   "repoUrl": "https://repo.maven.apache.org/maven2",
+  "tags": []
```

Comment:
Shouldn't the array contain the list in text form above?

Reply:
huuu.. yes of course...